plugins/emitevent.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: emitevent
+spec:
+  version: v0.0.2
+  homepage: https://github.com/rajatjindal/kubectl-emitevent
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/rajatjindal/kubectl-emitevent/releases/download/v0.0.2/kubectl-emitevent_v0.0.2_darwin_amd64.tar.gz
+    sha256: 330139310e94f83d767de79ec71ee2cc98f31227c727bf7b8e84c2a8729f1f36
+    bin: kubectl-emitevent
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/rajatjindal/kubectl-emitevent/releases/download/v0.0.2/kubectl-emitevent_v0.0.2_linux_amd64.tar.gz
+    sha256: 058a56a9dcf3ea919fb443fea8da89649056337c0fa347f39a669b47ae2f59ea
+    bin: kubectl-emitevent
+  shortDescription: Emit Kubernetes Events for the requested object
+  description: |
+    This plugin emits the Kubernetes Events for the requested object
```

Comment:
We normally don't recommend this anymore, but in this case I think it would be good to include an example command. This will give interested users a better idea how the plugin works. For example

> kubectl emit-event daemonset/kube-proxy -n kube-system --reason "foo-reason" --message "bar-message"